src/PrestaShopBundle/Form/Admin/Configure/ShopParameters/TrafficSeo/Meta/MetaType.php

```diff
@@ -112,13 +121,28 @@ public function buildForm(FormBuilderInterface $builder, array $options)
                                 'Admin.Notifications.Error'
                             ),
                         ]),
+                        new Length([
+                            'max' => SeoSettings::MAX_TITLE_LENGTH,
+                            'maxMessage' => $this->trans(
+                                'This field cannot be longer than %limit% characters',
+                                [
+                                    '%limit%' => SeoSettings::MAX_TITLE_LENGTH,
+                                ],
+                                'Admin.Notifications.Error'
+                            ),
+                        ]),
                     ],
-                    'required' => false,
                 ],
             ])
             ->add('meta_description', TranslatableType::class, [
                 'required' => false,
+                'type' => TextWithRecommendedLengthType::class,
                 'options' => [
+                    'required' => false,
```

Comment:
If it is false by default, you can remove this one.

Reply:
I like to make it not ambiguous 😄

Reply:
java developer.. :trollface:

Reply:
So I need to change it ?

Reply:
I think yes.

Reply:
Requested change has been applied and commit ammended